OpenEdX/View/MainScreenView.swift

```diff
@@ -188,7 +188,9 @@ struct MainScreenView: View {
         }
         .accentColor(Theme.Colors.accentXColor)
         .introspect(.viewController, on: .iOS(.v15)) { controller in
-            controller.navigationController?.setNavigationBarHidden(true, animated: false)
+            if viewModel.selection != .profile {
+                controller.navigationController?.setNavigationBarHidden(true, animated: false)
+            }
```

Comment:
@forgotvas  Is this change related to the mentioned issue?

Reply:
that change related to profile icon issue on profile tab on iOS 15 it disappears when you go to settings and then go back to profile. So just fixed in that PR too. Added to description.

Reply:
I guess the issue is also with the parsing, app is expecting, `"account_privacy": "private"` but server is returning `"account_privacy": "PRIVATE"`. 

Can you look into it under the same PR or should create a separate ticket for it. 

Credentials: `rtester97/Fedx1234`

Reply:
added in https://github.com/edx/edx-mobile-marketplace-ios/pull/59